### DISCUSSION TITLE
*Solved* "not a" Bug, GoPro 11 Mini 2.00.70, 1.10.70, "WAKE=0" function does not work as described

### DISCUSSION THREAD

**Discussion by EIIIE** [gopro/labs#192] [General]
When you enter "WAKE=0", the qr code is read, but it does not seem to be applied (since it is not reflected in the short list when the camera starts). After the camera reads the QR code "wake=0" and turns off, it will still turn on when power is supplied via USB.

![image](https://user-images.githubusercontent.com/13149979/211943991-cdb35d79-359b-4598-a9fd-ff6ec922f228.png)

**Comment by dnewman-gpsw** [comment]
Moved to discussions as this is working correctly.  wake=0 and WAKE=0 is not the same as the full command !MWAKE=0  Make you are using the full command.  I have test on 2.00.70, that the WAKE feature is working as documented (including disabling.)

**Reply by EIIIE** [reply]
When used by you "!MWAKE=0" is this command displayed when the camera is turned on?

I use the syntax from QRControl and judging by the command from below it is correct - "!MWAKE=0".

**Reply by dnewman-gpsw** [reply]
Nothing set to 0 will be displayed, as that is the off state, which it the default condition.
